src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -99,7 +99,7 @@ def _raise_error_if_not_detection_sframe(dataset, feature, annotations, require_
 
 def create(dataset, annotations=None, feature=None, model='darknet-yolo',
            classes=None, batch_size=0, max_iterations=0,
-           verbose=True, grid_shape=[13, 13], **kwargs):
+           verbose=True, grid_shape=[13, 13], random_seed=0, **kwargs):
```

Comment:
This is not how other public API accepting random seeds work. With this default, won't the toolkit produce the same behavior every time unless the user manually randomizes the seed?

Reply:
No they won't produce the same result.
For OD if random_seed is not provided, it is `FLEX_UNDEFINED` in C++,
and then a random seed is produced.
Don't know if this answer your question...

Reply:
Well, in DC, if the value is `FLEX_UNDEFINED`, the random seed will be 0, implicitly. I guess you can mention it in docstring about this implicit behavior and add similar logic in C++ implementation, in which way there's no need to change the C-API.

Reply:
https://github.com/apple/turicreate/blob/4e2d9159b27e321689c3aa6db43a0400eb98ad68/src/toolkits/drawing_classifier/drawing_classifier.cpp#L334

Reply:
Good!